shared/components/developer-resources-section.tsx

```diff
@@ -24,14 +24,15 @@ export const DeveloperResourcesSection: React.FC<DeveloperResourcesSectionProps>
 }) => {
   const { useTranslate } = useThemeHooks();
   const { translate } = useTranslate();
+  const isSingleCard = cards.length === 1;
 
   return (
-    <div className={`developer-resources-section page-community ${className}`}>
+    <div className={` container developer-resources-section page-community ${className} ${isSingleCard ? 'single-card' : ''}`}>
       <section className="bottom-cards-section bug-bounty section-padding">
         {cards.map((card, index) => (
-          <div key={index} className="com-card">
+          <div key={index} className={`com-card ${isSingleCard ? 'single-card' : ''}`}>
             <img
-              className={`${index === 0 ? 'top-right-img' : 'bottom-right-img'} bug-bounty-card-bg${index === 0 ? '' : '-2'} ${card.backgroundClass || ''}`}
+              className={`${isSingleCard ? 'bottom-right-img' : (index === 0 ? 'top-right-img' : 'bottom-right-img')} ${isSingleCard ? 'bug-bounty-card-bg-2' : `bug-bounty-card-bg${index === 0 ? '' : '-2'}`} ${card.backgroundClass || ''}`}
```

Comment:
Very minor nit: `bug-bounty` seems like an inappropriate name for this class, since on this page the section has nothing to do with a bug bounty.